consensus/polybft/polybft_test.go

```diff
@@ -264,3 +266,29 @@ func Test_Factory(t *testing.T) {
 	assert.Equal(t, epochSize, polybft.consensusConfig.EpochSize)
 	assert.Equal(t, params, polybft.config)
 }
+
+func Test_MakeKoskSignature(t *testing.T) {
+	expected := "127cfb8e2512b447056f33b91fca6cb2a7039e8b330edc4e5e5287f1c58bba5206373a97c9f09db144c8db5681c39e013ee6039ebbe36e0448e9f704f2d326c0"
+	bytes, _ := hex.DecodeString("3139343634393730313533353434353137333331343333303931343932303731313035313730303336303738373134363131303435323837383335373237343933383834303135343336383231")
+
+	pk, err := bls.UnmarshalPrivateKey(bytes)
+	require.NoError(t, err)
+
+	address := types.BytesToAddress((pk.PublicKey().Marshal())[:20])
```

Comment:
Is this suggestion correct?

```suggestion
	address := types.BytesToAddress((pk.PublicKey().Marshal())[:types.AddressLength])
```

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1184/commits/d282b8e0012ae417736023cb79f7f916ad08a38c